Common/src/CConfig.cpp

```diff
@@ -2371,7 +2371,7 @@ void CConfig::SetConfig_Options() {
   addDoubleOption("KNOWLES_B", Knowles_B, 1.0);
   /* DESCRIPTION: Knowles N constant */
   addDoubleOption("KNOWLES_N", Knowles_N, 1.0);
-
+  
```

Comment:
Be sure to run clang format before pushing in

Reply:
We don't clang-format the config yet...